src/Toolkit/Toolkit.Maui/Internal/HtmlToView.Maui.cs

```diff
@@ -0,0 +1,430 @@
+                {
+                    colSpan = colSpanFromAttr;
+                    Grid.SetColumnSpan(cellView, colSpan);
+                }
+                if (attr.TryGetValue("rowspan", out var rowSpanStr) && ushort.TryParse(rowSpanStr, out var rowSpanFromAttr))
+                {
+                    rowSpan = rowSpanFromAttr;
+                    Grid.SetRowSpan(cellView, colSpan);
+                }
+                gridView.Add(cellView, curCol, curRow);
+
+                // Mark grid-cells occupied by the current table-cell
+                for (int i = 0; i < rowSpan; i++)
+                {
+                    for (int j = 0; j < colSpan; j++)
+                    {
+                        EnsureColumnExists(curRow + i, curCol + j);
+
+                        gridMap[curRow + i][curCol + j] = true;
+
+                        maxRowUsed = Math.Max(maxRowUsed, curRow + i);
+                        maxColUsed = Math.Max(maxColUsed, curCol + j);
+                    }
+                }
+                curCol += colSpan;
+            }
+            curRow++;
+        }
+
+        // Now we know exactly how many rows and columns were necessary to hold the table. Allocate them!
+        for (int i = 0; i <= maxRowUsed; i++)
+            gridView.RowDefinitions.Add(new RowDefinition());
+        for (int i = 0; i <= maxColUsed; i++)
+            gridView.ColumnDefinitions.Add(new ColumnDefinition());
+
+        return gridView;
+
+        // Expand the gridMap as needed to make sure that given row/column exists
+        void EnsureColumnExists(int row, int col)
+        {
+            while (gridMap.Count <= row)
+                gridMap.Add(new List<bool>());
+            while (gridMap[row].Count <= col)
+                gridMap[row].Add(false);
+        }
+    }
+
+    private static void ApplyStyle(Span el, MarkupNode node)
+    {
+        if (node.IsBold == true)
+            el.FontAttributes |= FontAttributes.Bold;
+        else if (node.IsBold == false)
+            el.FontAttributes &= ~FontAttributes.Bold;
+
+        if (node.IsItalic == true)
+            el.FontAttributes |= FontAttributes.Italic;
+        else if (node.IsItalic == false)
+            el.FontAttributes &= ~FontAttributes.Italic;
+
+        if (node.IsUnderline == true)
+            el.TextDecorations |= TextDecorations.Underline;
+        else if (node.IsUnderline == false)
+            el.TextDecorations &= ~TextDecorations.Underline;
+
+        if (node.FontColor.HasValue)
+            el.TextColor = ConvertColor(node.FontColor.Value);
+        if (node.BackColor.HasValue)
+            el.BackgroundColor = ConvertColor(node.BackColor.Value);
+        if (node.FontSize.HasValue)
+            el.FontSize = 16d * node.FontSize.Value; // based on AGOL's default font size
+
+        if (node.IsMonospace.HasValue && node.IsMonospace.Value)
+            el.FontFamily = "Courier New";
```

Comment:
If you want to match Calcite's font choice on all platforms, consider complicating this a bit:

```suggestion
        if (node.IsMonospace.HasValue && node.IsMonospace.Value)
        {
            el.FontFamily = DeviceInfo.Platform switch
            {
                DevicePlatform.Android     => "monospace",
                DevicePlatform.iOS         => "Menlo",
                DevicePlatform.MacCatalyst => "Monaco",
                _                          => "Consolas"
            };
        }
```

Reply:
This isn't a calcite library. "Courier New" works on all platforms and is part of the standard set of fonts any device has past, present and future. For consistency I think it's better to stick to that.